qa/suites/fs/upgrade/featureful_client/upgraded_client/tasks/2-upgrade.yaml

```diff
@@ -20,10 +20,9 @@ tasks:
 - mds_pre_upgrade:
 - print: "**** done mds pre-upgrade sequence"
 - install.upgrade:
+    # upgrade the single cluster node, which running the mon/mds/osd/mgr daemons
     mon.a:
       branch: quincy
-    mon.b:
-      branch: quincy
 - print: "**** done install.upgrade both hosts"
```

Comment:
Now that it's a single node, let's change this sentence ?

Reply:
Done. Thanks!